src/main/java/com/google/sps/image/ImageScorer.java

```diff
@@ -35,6 +37,15 @@ public ImageScorer(String[] keywords) {
     this.keywords = keywords;
   }
 
+  private Map<String, Boolean> generateMap(List<String> elements) {
```

Comment:
`Iterable<String> elements`

would also work, you really just need to be able to iterate over the collection when building the map.

Reply:
Done.